peachjam/templates/peachjam/layouts/main.html

```diff
@@ -20,8 +20,8 @@
         Sentry.init({
         dsn: '{{ SENTRY_DSN_KEY }}',
         environment: '{{ SENTRY_ENVIRONMENT }}',
-        blacklistUrls: [],
-        });
+        whitelistUrls: [`${window.location.origin}/static/js/app-prod.js`]
```

Comment:
This needs to be a regex. In production, the app-prod.js is fingerprinted with a hash. I'd say anything that is included from static/js we should know about.

Reply:
done